WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -0,0 +1,546 @@
+import Foundation
+import UIKit
+import SVProgressHUD
+import WordPressShared
+import WordPressComAnalytics
+
+/// A view controller that presents a Jetpack login form
+///
+class JetpackLoginViewController: UIViewController {
+
+    // MARK: - Constants
+
+    fileprivate let jetpackInstallRelativePath = "plugin-install.php?tab=plugin-information&plugin=jetpack"
+    fileprivate let jetpackMoreInformationURL = "https://apps.wordpress.com/support/#faq-ios-15"
```

Comment:
This is the right URL but I'm noticing in the web view the browser isn't positioned properly at question 25:

![simulator screen shot feb 1 2017 4 29 37 pm](https://cloud.githubusercontent.com/assets/373903/22529092/ae39a6f0-e89b-11e6-9a59-1e66c047c5a3.png)

Not sure there's much we can do about it here.